google-cloud-debugger/lib/google/cloud/debugger/rails.rb

```diff
@@ -58,19 +58,20 @@ class Railtie < ::Rails::Railtie
         end
 
         initializer "Stackdriver.Debugger" do |app|
-          debugger_config = Railtie.parse_rails_config config
+          if self.class.use_debugger? app.config
+            debugger_config = Railtie.parse_rails_config config
```

Comment:
This `config` is defined in `railties` gem: https://github.com/rails/rails/blob/master/railties/lib/rails/railtie.rb#L122

The `config` and `app.config` are interchangeable in this initializer code. They both reference the same object. I actually thought about changing it here from `config` to `app.config`, but then I noticed the same code is referenced in the other three Railties we have. (This code is probably inherited from the previous `appengine` gem)